frontend/components/app.jsx

```diff
@@ -4,14 +4,16 @@ import { AuthRoute, ProtectedRoute } from '../util/route_util';
 import Landing from 'components/landing';
 import Home from 'components/home';
 import SessionModal from 'components/session_modal';
+import TaskModal from 'components/task_modal';
 
 const App = () => {
   return (
     <section id="app-container">
       <Route exact path="/" component={Landing}/>
       <AuthRoute path="/login" component={SessionModal} />
       <AuthRoute path="/signup" component={SessionModal} />
-      <ProtectedRoute exact path="/home" component={Home} />
+      <ProtectedRoute path="/home" component={Home} />
+      <TaskModal />
```

Comment:
minor - i would expect this render in the `<Home/>` component
also confusing - `SessionModal` is more of a page not a true modal